### ISSUE TITLE
VM Tag

### ISSUE THREAD

**Issue by Intybyte** [xmrig/xmrig#3362]
At first i had some problems in making MSR work on win11, but after disabling core isolation it works and it was applied correctly, however xmrig still shows the red VM tag, does it lower my hashrate? If yes how do I fix it? Thank you

**Comment by Intybyte** [comment]
Disabling Virtualization on cpu fixed it but is there a work around? I need virtualization on if possible

**Comment by geekwilliams** [comment]
Does having virtualization on actually effect your hashrate? I'm curious because I'm in the same position

**Comment by Intybyte** [comment]
Maybe a 50-100 H/s gain? Or maybe none, it is a small difference if there is some

**Comment by geekwilliams** [comment]
Makes sense. I run vm's on my workstation, that also mines when I'm not using it. 50-100h/s doesn't matter that much to me.  I wouldn't think virtualization would effect even the msr mod but I don't know enough about it to say for sure

**Comment by SlavisaBakic** [comment]
> At first i had some problems in making MSR work on win11, but after disabling core isolation it works and it was applied correctly, however xmrig still shows the red VM tag, does it lower my hashrate? If yes how do I fix it? Thank you

Is Hyper-V enabled?

**Comment by Intybyte** [comment]
> > At first i had some problems in making MSR work on win11, but after disabling core isolation it works and it was applied correctly, however xmrig still shows the red VM tag, does it lower my hashrate? If yes how do I fix it? Thank you
> 
> Is Hyper-V enabled?

In order to disable virtualization you need to disable Hyper-V, so it isn't enabled

**Comment by SlavisaBakic** [comment]
> > > At first i had some problems in making MSR work on win11, but after disabling core isolation it works and it was applied correctly, however xmrig still shows the red VM tag, does it lower my hashrate? If yes how do I fix it? Thank you
> > 
> > 
> > Is Hyper-V enabled?
> 
> In order to disable virtualization you need to disable Hyper-V, so it isn't enabled

Open System Information in Windows and check there if Virtualization Based Security is enabled.

**Comment by Intybyte** [comment]
> > > > At first i had some problems in making MSR work on win11, but after disabling core isolation it works and it was applied correctly, however xmrig still shows the red VM tag, does it lower my hashrate? If yes how do I fix it? Thank you
> > > 
> > > 
> > > Is Hyper-V enabled?
> > 
> > In order to disable virtualization you need to disable Hyper-V, so it isn't enabled
> 
> Open System Information in Windows and check there if Virtualization Based Security is enabled.

It is disabled

**Comment by SlavisaBakic** [comment]
> > > > > At first i had some problems in making MSR work on win11, but after disabling core isolation it works and it was applied correctly, however xmrig still shows the red VM tag, does it lower my hashrate? If yes how do I fix it? Thank you
> > > > 
> > > > 
> > > > Is Hyper-V enabled?
> > > 
> > > 
> > > In order to disable virtualization you need to disable Hyper-V, so it isn't enabled
> > 
> > 
> > Open System Information in Windows and check there if Virtualization Based Security is enabled.
> 
> It is disabled

In Windows Features Turn Off and On dialog make sure that those features are disabled:

- Hyper-V
- Microsoft Defender Application Guard
- Virtual Machine Platform
- Windows Hypervisor Platform
- Windows Sandbox
- Windows Subsystem for Linux
